src/app/settings/components/project-role-jwt-tokens/project-role-jwt-tokens.scss

```diff
@@ -0,0 +1,6 @@
+.hideToken {
```

Comment:
Please rename to `.project-role-jwt-tokens__hide-token` . We use BEM to avoid CSS rules naming collisions